build/onos-it/Dockerfile

```diff
@@ -4,8 +4,8 @@ FROM onosproject/onos-config-base:$ONOS_CONFIG_BASE_VERSION as base
 
 FROM alpine:3.8
 
-COPY --from=base /go/src/github.com/onosproject/onos-config/build/_output/onit /usr/local/bin/onit
+COPY --from=base /go/src/github.com/onosproject/onos-config/build/_output/onit-k8s /usr/local/bin/onit-k8s
 
 USER nobody
 
-ENTRYPOINT ["onit", "test"]
```

Comment:
why remove "test"

Reply:
onit-k8s has two options "test" or "suite" that trigger either a single test or a suite. No need to put the command one more "level" down since it's its own command now and would not interfere with any onit commands